descwl_shear_sims/sim.py

```diff
@@ -52,12 +52,13 @@ def make_sim(
     rng,
     galaxy_catalog,
     coadd_dim,
-    g1,
-    g2,
+    shear_obj,
```

Comment:
Can we make this somehow backwards compatible?

Reply:
Done, see [this unit test](https://github.com/mr-superonion/descwl-shear-sims/blob/shear_obj_zshear/shear_meas_tests/test_shear_meas_zshear_metacal_compatible.py) for backward compatibility!